Parameters:
    aFirstYamlParam:  'firstParam'
    a2ndYamlParam:  'second'
